src/Training.TruckWorld.Clone.Backend/Training.TruckWorld.Backend/Application/Emails/Interfaces/IEmailSenderService.cs

```diff
@@ -0,0 +1,8 @@
+﻿using Training.TruckWorld.Backend.Domain.Entities;
+
+namespace Training.TruckWorld.Backend.Application.Emails.Interfaces;
+
+public interface IEmailSenderService
+{
+    ValueTask<Email> SendEmailAsync(Email email);
```

Comment:
bo''ta ValueTask<bool> bo'miydimi?

Reply:
ya'ni email ni o'rniga bool bo'miydimi ValueTaskni ichida

Reply:
To'g'ri ,bool bo'ladi o'zgartirib qo'yaman